src/layouts/VerticalLayout.tsx

```diff
@@ -1,7 +1,7 @@
 import { LayoutProps, GroupLayout } from "@jsonforms/core"
 import { AntDLayout, AntDLayoutProps } from "./LayoutRenderer"
 import { Form, FormInstance, FormProps } from "antd"
-import { VerticalLayout as VerticalLayoutUISchema } from "../ui-schema"
+import { VerticalLayoutUISchema } from "../ui-schema"
```

Comment:
That we landed on a name that was already used in the codebase as an alias means we've picked the "[desire type](https://en.wikipedia.org/wiki/Desire_path)"